src/index.js

```diff
@@ -118,12 +122,18 @@ const run = async () => {
 			if (hasChanges === false && modified.length < 1) {
 				core.info('File(s) already up to date')
 
-				if (existingPr) await git.removePrWarning()
+				if (SKIP_PR) {
```

Comment:
This check is not really necessary, as the PR warning only gets removed if there is an existing PR.